src/com/yetanalytics/lrs_admin_ui/views/update_password.cljs

```diff
@@ -0,0 +1,70 @@
+(ns com.yetanalytics.lrs-admin-ui.views.update-password
+  (:require [reagent.core :as r]
+            [re-frame.core :refer [subscribe dispatch]]
+            [com.yetanalytics.lrs-admin-ui.functions :as fns]
+            [com.yetanalytics.lrs-admin-ui.functions.copy :refer [copy-text]]
+            [com.yetanalytics.lrs-admin-ui.functions.password :as pass]
+            [com.yetanalytics.lrs-admin-ui.input :refer [p-min-len]]
+            [goog.string :refer [format]]
+            goog.string.format))
+
+(defn form []
+  (let [hide-pass (r/atom true)]
+    (fn []
+      (let [new-password @(subscribe [:update-password/new-password])]
+        [:div {:class "create-account-inputs"}
+         [:div {:class "row"}
+          [:label {:for "old-password-input"} "Old Password:"]
+          [:input {:value @(subscribe [:update-password/old-password])
+                   :class "new-account round"
+                   :id "old-password-input"
+                   :type "password"
+                   :on-change #(dispatch [:update-password/set-old-password
+                                          (fns/ps-event-val %)])}]]
+         [:div {:class "row pt-2"}
+          [:label {:for "new-password-input"} "New Password:"]
+          [:input (cond-> {:value new-password
+                           :class "new-account round"
+                           :id "new-password-input"
+                           :on-change #(dispatch [:update-password/set-new-password
+                                                  (fns/ps-event-val %)])}
+                    @hide-pass (merge {:type "password"}))]
+          [:ul {:class "action-icon-list"}
+           [:li
+            [:a {:href "#!",
+                 :class "icon-secret pointer"
+                 :on-click #(swap! hide-pass not)}
+             (str (cond
+                    @hide-pass "Show"
+                    :else "Hide"))]]
```

Comment:
Why not a one-liner `if` statement?
```clojure
(if @hide-pass "Show" "Hide")
```

Reply:
because I copy-pasted this from the other form where it does this